<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Freeze</class>
 <widget class="QWidget" name="Freeze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="query_state">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>状态查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_id">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卡号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_mode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>冻结/解冻</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="need_admin_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>88</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>挂失模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>校园卡处理状态</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
